src/main/java/teammates/ui/webapi/GateKeeper.java

```diff
@@ -111,7 +35,7 @@ public void verifyLoggedInUserPrivileges() {
     /**
      * Verifies that the specified student can access the specified course.
      */
-    public void verifyAccessible(StudentAttributes student, CourseAttributes course) {
+    void verifyAccessible(StudentAttributes student, CourseAttributes course) {
```

Comment:
Any particular reason for removing access modifiers? If it's meant to be package private we could go with `protected` instead of using the default access modifier.

Reply:
Isn't package default more restrictive? `protected` allows package default plus access to child classes, but `GateKeeper` is not meant to have any.

Reply:
Ah okay yes you are right